showcase/src/main/kotlin/com/recco/showcase/main/MainScreen.kt

```diff
@@ -38,94 +44,83 @@ fun MainScreen(
-                contentColor = SoftYellow
-            ),
-            shape = RoundedCornerShape(0.dp),
-            onClick = openReccoClick
-        ) {
-            Text(
-                text = stringResource(id = R.string.open_recco),
-                style = Typography.displayMedium
-            )
-        }
+            Button(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(48.dp),
+                colors = ButtonDefaults.filledTonalButtonColors(
+                    containerColor = WarmBrown,
+                    contentColor = SoftYellow
+                ),
+                shape = RoundedCornerShape(0.dp),
+                onClick = openReccoClick
+            ) {
+                Text(
+                    text = stringResource(id = R.string.open_recco),
+                    style = Typography.displayMedium
+                )
+            }
 
-        Button(
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(top = 16.dp)
-                .height(48.dp),
-            colors = ButtonDefaults.outlinedButtonColors(
-                contentColor = WarmBrown
-            ),
-            shape = RoundedCornerShape(0.dp),
-            border = BorderStroke(width = 1.dp, WarmBrown),
-            onClick = navigateToCustomizeScreen
-        ) {
-            Text(
-                text = stringResource(id = R.string.customize),
-                style = Typography.displayMedium.copy(color = WarmBrown)
-            )
+            Button(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 16.dp)
+                    .height(48.dp),
+                colors = ButtonDefaults.outlinedButtonColors(
+                    contentColor = WarmBrown
+                ),
+                shape = RoundedCornerShape(0.dp),
+                border = BorderStroke(width = 1.dp, WarmBrown),
+                onClick = logoutClick
+            ) {
+                Text(
+                    text = stringResource(id = R.string.logout),
+                    style = Typography.displayMedium.copy(color = WarmBrown)
+                )
+            }
         }
 
-        Button(
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(top = 16.dp)
-                .height(48.dp),
-            colors = ButtonDefaults.outlinedButtonColors(
-                contentColor = WarmBrown
-            ),
-            shape = RoundedCornerShape(0.dp),
-            border = BorderStroke(width = 1.dp, WarmBrown),
-            onClick = logoutClick
-        ) {
-            Text(
-                text = stringResource(id = R.string.logout),
-                style = Typography.displayMedium.copy(color = WarmBrown)
-            )
-        }
+        val application = LocalContext.current.applicationContext as Application
+        PaletteSelection(selectionClickPalette = { reccoPalette ->
```

Comment:
Does it make sense to hold a state in which default palette is referenced ? With the current design, at least IMHO it is not clear which is the current selected palette.

Reply:
Let's discuss it during the call, I did not find other way to achieve this.